addons/volumetric_importer/scripts/Editor/TransferFunctionInspector.cs

```diff
@@ -0,0 +1,42 @@
+#if TOOLS
+using Godot;
+
+namespace VolumetricRendering
+{
+    public partial class TransferFunctionInspector : EditorInspectorPlugin
+    {
+        // This is a hack to make sure we can show the histogram in the TF editor.
+        // TODO: Can we get the "parent" of the TransferFunctionInspector instead?
```

Comment:
You mean in order to put the "Edit transfer function" button in the VolumeRenderedObject Inspector?

Reply:
So there are two ways to edit transfer functions:
1. Edit a transfer function resource asset (not tied to a volume) => you won't see a histogram
2. Edit a transfer function through the VolumeRenderedObject inspector

The same TransferFunctionInspector will automatically be used in both cases.

For the 2nd case, the TransferFunctionInspector needs to know which VolumeRenderedObject we're editing the TF of, so it can render its histogram in the background. It would be nicer if the TransferFunctionInspector could figure this out on its own (by getting its "parent" inspector), but I don't think that's possible.

We could solve that by putting an "Edit transfer function" button directly in the VolumeRenderedObject inspector, but we also need to have that button in the TransferFunctionInspector inspector anyway (so we can edit TFs saved as assets - not through a VolumeRenderedObject ).

Sorry if it sounds confusing 😅  I'll keep investigating. But this works fine for now :)